cogs5e/gametrack.py

```diff
@@ -554,12 +554,29 @@ async def customcounter_delete(self, ctx, name):
         await ctx.send(f"Deleted counter {counter.name}.")
 
     @customcounter.command(name='summary', aliases=['list'])
-    async def customcounter_summary(self, ctx):
-        """Prints a summary of all custom counters."""
+    async def customcounter_summary(self, ctx, page: int = 0):
+        """Prints a summary of all custom counters. Will use paging if more than 25."""
```

Comment:
```suggestion
        """
        Prints a summary of all custom counters.
        Use `!cc list <page>` to view pages if you have more than 25 counters.
        """
```

The first line of the docstring is considered the "short" summary (i.e. what appears as the command description in `!help`, and what's displayed without opening the command on .io/commands)